pkg/objectio/fs.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objectio
+
+import (
+	"fmt"
+	"github.com/matrixorigin/matrixone/pkg/fileservice"
+	"sync"
+)
+
+type ObjectFS struct {
+	sync.RWMutex
+	Service fileservice.FileService
+	Dir     string
+	Writer  map[string]Writer
+}
+
+func TmpNewFileservice(dir string) fileservice.FileService {
+	c := fileservice.Config{
+		Name:    "LOCAL",
+		Backend: "DISK",
+		DataDir: dir,
+	}
+	service, err := fileservice.NewFileService(c)
+	if err != nil {
+		panic(any(fmt.Sprintf("NewFileService failed: %s", err.Error())))
```

Comment:
Why don't you panci(err)?

Reply:
This is a temporary function, fileservice needs to be initialized in main.go and then passed in, so there is no need to create fileservice here

Reply:
We should always panci a moerr.